src/components/Skill/Skill.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Box as MuiBox, BoxProps as MuiBoxProps } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+type TLevelRange = 0 | 1 | 2 | 3 | 4 | 5;
+
+export interface SkillProps extends MuiBoxProps {
```

Comment:
Omit the `children` prop from the `MuiBox` component or use it instead of the `label` prop. Also, change the interface to type